custom_components/climate_group/climate.py

```diff
@@ -320,8 +320,16 @@ def async_update_group_state(self) -> None:
         # so that we don't break in the future when a new feature is added.
         self._attr_supported_features &= SUPPORT_FLAGS
 
+        self._enable_turn_on_off_backwards_compatibility = False
+
         _LOGGER.debug("State update complete")
 
+    async def async_turn_off(self) -> None:
+        await self.async_set_hvac_mode(HVACMode.OFF)
+
+    async def async_turn_on(self) -> None:
```

Comment:
@bjrnptrsn / @ladzar This breaks Climate turn on and turn off by forcibly overriding the aync_turn_on mode to HVACMode heat.  I am using this for cool only air conditioners and as such this causes them to fail to turn on as an unsupported mode with the generic climate turn on service, breaking google assistant capability.

Reply:
Please revert this change and come up with a better approach.  Such as just relying on the default mode which is whatever the entity last HVACMode was when toggled.

Reply:
I think that generic turn on/off is not really a good idea, it is not a light switch or a socket.  The climate entities should be called with Climate turn on service that takes the HVAC Mode.

Reply:
Hi, yeah you're right.... I'm using it to control heaters only and actually this didn't occur to me. Before the change your climate group worked even with Google Assistant? Mine didn't, that's why I modified it in the first place.

Reply:
@ladzar It worked for me with the default aync_turn_on because my climate devices store the last HVAC mode for turn on.  It looks like your climate devices don't which is why you need to force a mode.